packages/priority-queue/README.md

```diff
@@ -0,0 +1,29 @@
+# Priority Queue
+
+This module allows you to run a queue of callback while on the browser's idle time making sure the higher-priority work is performed before.
```

Comment:
Grammar: I think "first", "earlier", or providing some "before _what_" would read better at the end of this sentence.